src/modules/CC1101/CC1101.cpp

```diff
@@ -351,7 +414,12 @@ int16_t CC1101::setBitRate(float br) {
   // set bit rate value
   int16_t state = SPIsetRegValue(CC1101_REG_MDMCFG4, e, 3, 0);
   state |= SPIsetRegValue(CC1101_REG_MDMCFG3, m);
-  return(state);
+
+  if (state == ERR_NONE) {
+    _br = br;
```

Comment:
CI is complaining `_br` doesn't exist - I guess it's not declared in the header.